test/ig/IGExporter.IG.test.ts

```diff
@@ -3119,4 +3119,263 @@ describe('IGExporter', () => {
       ]);
     });
   });
+
+  describe('#r5-properties-on-r4-igs', () => {
+    let tempOut: string;
+
+    beforeEach(() => {
+      loggerSpy.reset();
+      tempOut = temp.mkdirSync('sushi-test');
+      const fixtures = path.join(__dirname, 'fixtures', 'simple-ig');
+      const defs = new FHIRDefinitions();
+      // r5-definitions contains the guide-parameter-code CodeSystem, which was originally included in 5.0.0-ballot
+      loadFromPath(path.join(__dirname, '..', 'testhelpers', 'testdefs'), 'r5-definitions', defs);
+
+      const r4WithR5propsConfig = cloneDeep(minimalConfig);
+      r4WithR5propsConfig.copyrightLabel = 'Shorty Fsh 2022+';
+      r4WithR5propsConfig.dependencies = [
+        {
+          packageId: 'hl7.fhir.us.ocean',
+          uri: 'http://example.org/ocean',
+          id: 'ocean',
+          version: '1.0.0',
+          reason: 'The ocean is essential because it has fsh'
+        }
+      ];
+      r4WithR5propsConfig.resources = [
+        {
+          reference: { reference: 'Patient/patient-example' },
+          profile: [
+            'http://example.org/StructureDefiniton/first',
+            'http://example.org/StructureDefiniton/second'
+          ]
+        }
+      ];
+      r4WithR5propsConfig.pages = [
+        {
+          nameUrl: 'index.md',
+          title: 'Home',
+          generation: 'markdown',
+          sourceMarkdown: 'source markdown for index'
+        },
+        {
+          nameUrl: 'other-page.md',
+          title: 'Some Other Page',
+          generation: 'markdown',
+          sourceUrl: 'http://example.org',
+          page: [
+            {
+              nameUrl: 'nested-page.md',
+              sourceString: 'source string for nested page'
+            },
+            {
+              nameUrl: 'second-nested-page.md'
+            }
+          ]
+        }
+      ];
+
+      const pkg = new Package(r4WithR5propsConfig);
+
+      // Example: Patient/patient-example
+      const instanceDef3 = InstanceDefinition.fromJSON(
+        fs.readJSONSync(path.join(fixtures, 'examples', 'Patient-example.json'))
+      );
+      instanceDef3._instanceMeta.usage = 'Example';
+      pkg.instances.push(instanceDef3);
+
+      const exporter = new IGExporter(pkg, defs, fixtures);
+      // No need to regenerate the IG on every test -- generate it once and inspect what you
+      // need to in the tests
+      exporter.export(tempOut);
```

Comment:
Ah good call. I'll update this to a `beforeAll`, although the content may change based on your other comment.

Reply:
Updated in [6b5d6f2](https://github.com/FHIR/sushi/pull/1189/commits/6b5d6f2d87ae6ad596c2bb3fd0fbd955afa74455) but it might change again if I add more tests.